### DISCUSSION TITLE
(Feature Request) - sort lora

### DISCUSSION THREAD

**Discussion by showerstomp** [BlafKing/sd-civitai-browser-plus#121] [Ideas]
Problem: Lots of loras! webui doesn't auto detect properly!
Possible to add option to "sort" Lora's into subfolders 1.5, XL, unknown via hash matching with citivai database and lora model page during update scan?

**Comment by BlafKing** [comment]
I'm not exactly sure what you're asking? 😅 

What do you mean by "webui doesn't auto detect properly"?

I'm also not sure where you'd like to models to be sorted, do you mean sort them inside the WebUI per base model?
(Like how models can be sorted by date?)
Or would you like to be able to have a sub folder option named after the base model version of the model?

> unknown via hash matching with citivai database and lora model page during update scan?

If you're describing an error here, then please feel free to create a bug report! :)

**Reply by showerstomp** [reply]
Just the option to have lora models moved into subfolders 1.5/XL etc that match what model the database says they are for or unknown/notmoved if they dont match a known hash/model page

**Reply by BlafKing** [reply]
I'll add an option to save the models in a sub folder named by their base model version, I'll look into the possibility of adding an option to sort already existing models in the future!

**Reply by showerstomp** [reply]
Thank you! happy new year!

**Comment by BlafKing** [comment]
A sub folder option based on the models Base Model has now been added in v3.3.0! :)
